modules/olm-operatorgroups-limitations.adoc

```diff
@@ -0,0 +1,11 @@
+[id="olm-operatorgroups-limitations"]
+= Limitations for multi-tenant Operator management
+
+OpenShift Container Platform provides limited support for installing different variations of an Operator on a cluster at the same time. Operators are control plane extensions. Since all tenants, or namespaces, share the same control plane of a cluster, tenants in a multi-tenant environment also have to share Operators
```

Comment:
I was deliberating about the wording here, particularly the group of phrases at the end, "of an operator" "on a cluster" "at the same time". I'm wondering if this can be condensed a little for clarity, like: OpenShift Container Platform provides limited support for simultaneously installing different variations of an Operator on a cluster. Just a thought, I don't think there's anything wrong with the way you have it :)

Reply:
Sorry, also, I believe that last sentence may be a hair too long. Maybe bust it up into 2 sentences, All tenants, or namespaces, share the same control plane...cluster. Therefore, tenants....also share Operators.